client/remote_access.go

```diff
@@ -28,20 +28,29 @@ const (
 	UDP = "udp"
 )
 
+// RemoteAccessConnection defines a remote access connection.
+type RemoteAccessConnection struct {
+	// DeviceID is the device ID of the device to which the remote access connection belongs.
+	DeviceID string `json:"device_id"`
+
+	// Targets is a list of remote access targets.
+	Targets []string `json:"targets"`
+}
+
 // RemoteAccessTarget defines
 type RemoteAccessTarget struct {
 	// Protocol is the protocol used for the remote access.
 	// Can be either "tcp" or "udp".
-	Protocol string
+	Protocol string `json:"protocol"`
```

Comment:
As we are not parsing this field from a JSON file, we probably don't need the `json` tag, right?

Reply:
Done